supervised/automl.py

```diff
@@ -403,44 +397,44 @@ def __init__(
         self.random_state = random_state
 
     def fit(
-        self,
-        X: Union[numpy.ndarray, pandas.DataFrame],
-        y: Union[numpy.ndarray, pandas.Series],
-        sample_weight: Optional[Union[numpy.ndarray, pandas.Series]] = None,
-        cv: Optional[Union[Iterable, List]] = None,
-        sensitive_features: Optional[
-            Union[numpy.ndarray, pandas.Series, pandas.DataFrame]
-        ] = None,
+            self,
+            X: Union[np.ndarray, pd.DataFrame],
```

Comment:
why you changed `numpy` to `np` and `pandas` to `pd`?

Reply:
These aliases have become standardized within the Python data science community, so you'll often see them used in tutorials, documentation, and code examples. It's not a mandatory change, but it's a widely accepted convention that helps improve code readability and efficiency.